internal/nix/profile.go

```diff
@@ -216,14 +216,15 @@ func ProfileInstall(args *ProfileInstallArgs) error {
 	cmd.Env = AllowUnfreeEnv()
 	cmd.Args = append(cmd.Args, ExperimentalFlags()...)
 	cmd.Args = append(cmd.Args, args.ExtraFlags...)
-	cmd.Stdout = &PackageInstallWriter{args.Writer}
```

Comment:
Aside from ignoring some lines (which are no longer needed), I think we also needed this writer to indent the output:
> _, err = io.WriteString(fw.Writer, "\t"+line+"\n")